lib/commands/deploynodeapp.js

```diff
@@ -17,9 +21,13 @@ var ziputils = require('../ziputils');
 var parseDeployments = require('./parsedeployments');
 var fetchproxy = require('./fetchproxy');
 var deployproxy = require('./deployproxy');
+var templates = require('./edgeServicesTemplates');
 
 var DeploymentDelay = 60;
 var ProxyBase = 'apiproxy';
+var TriremeRuntimeOption = 'trireme';
+var EdgeServicesRuntimeOption = 'edgeServices'
+var DefaultNodeVersion = '7'
```

Comment:
We should use an [LTS](https://github.com/nodejs/LTS) version like `4.x` or `6.x`.  I'd almost say use the newest version that is in LTS, which would be `6.x`.